tests/test_envbuilder.py

```diff
@@ -278,6 +279,7 @@ def setUp(self):
         self.temp_folder = tempfile.mkdtemp()
         self.file_path = os.path.join(self.temp_folder, 'usage_stats')
         self.addCleanup(lambda: os.path.exists(self.tempfile) and os.remove(self.tempfile))
```

Comment:
Why do we keep the cleanup of inside files? Isn't removing the whole dir enough?

Reply:
check the first line of `setUp` the test is creating two temp files.